Sources/Extensions/MKMapView+PinFloyd.swift

```diff
@@ -0,0 +1,57 @@
+import MapKit
+
+// MARK: - Tile bounding box
+
+struct MapTileBoundingBox {
```

Comment:
Is this the same as this bounding box http://wiki.openstreetmap.org/wiki/Bounding_Box?